packages/suite/src/views/wallet/coinmarket/sell/offers/Offers/List/index.tsx

```diff
@@ -100,14 +101,18 @@ const List = ({ isAlternative, quotes }: Props) => {
                 <Left>
                     <>
                         <SummaryRow>
-                            {amountInCrypto && (
+                            <StyledCoinLogo size={21} symbol={account.symbol} />
+                            {amountInCrypto ? (
                                 <Send>
-                                    {formatCryptoAmount(Number(quotes[0].cryptoStringAmount))}
+                                    <FormattedCryptoAmount
+                                        value={quotes[0].cryptoStringAmount}
+                                        symbol={account.symbol}
+                                    />
                                 </Send>
+                            ) : (
+                                <Crypto>{quotes[0].cryptoCurrency}</Crypto>
```

Comment:
It's confusing, why is a component named `Crypto` rendered when `amountInCrypto === false`? 😅